### PR TITLE
Fixed: OpenSSL hashing review comments.

### PR DESCRIPTION
**Pull request by pkthapa**
Review comments from below PR addressed:

1. https://github.com/arc-vmware/concord-bft/pull/3

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -114,9 +114,9 @@ if(CODECOVERAGE)
     message( "-- Building with llvm Code Coverage Tools")
 endif()
 
-if(USE_CRYPTOPP_HASH)
-    message("-- USE_CRYPTOPP_HASH Enabled")
-    string(APPEND CMAKE_CXX_FLAGS " -DUSE_CRYPTOPP_HASH")
+if(USE_CRYPTOPP_SHA_256)
+    message("-- USE_CRYPTOPP_SHA_256 Enabled")
+    string(APPEND CMAKE_CXX_FLAGS " -DUSE_CRYPTOPP_SHA_256")
```
Comment (toly-kournik) [inline CMakeLists.txt]: please move it to util/CMakeLists.txt and use 
`target_compile_definitions(util PUBLIC USE_CRYPTOPP_SHA_256)`

**Comment by toly-kournik** [comment]
@pkthapa , there are many unrelated changes in your PR, please rebase on master to isolate only your changes.

File: util/include/openssl_digest_creator.ipp [hunk 2]
```diff
@@ -12,19 +12,18 @@
 #pragma once
```
Comment (toly-kournik) [inline util/include/openssl_digest_creator.ipp:12]: Why is this file named *.ipp?

Comment (pkthapa) [reply util/include/openssl_digest_creator.ipp:12]: Resolved in https://github.com/arc-vmware/concord-bft/pull/5/commits/0e56648d6e0c220038d05b247a4bc589806a90a9

File: CMakeLists.txt (hunk 1 above)
Comment (pkthapa) [reply CMakeLists.txt]: Resolved in https://github.com/arc-vmware/concord-bft/pull/5/commits/0e56648d6e0c220038d05b247a4bc589806a90a9

**Review by arc-vmware — APPROVED** [review]

**Pull request merged**